cmd/cli/commands/tokens.go

```diff
@@ -0,0 +1,37 @@
+package commands
+
+import (
+	"os"
+
+	"github.com/sonm-io/core/blockchain"
+	"github.com/sonm-io/core/blockchain/tsc"
+	"github.com/sonm-io/core/util"
+	"github.com/spf13/cobra"
+)
+
+var approveTokenCmd = &cobra.Command{
+	Use:    "approve <amount>",
+	Short:  "Approve tokens",
+	PreRun: loadKeyStoreWrapper,
+	Args:   cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		bch, err := blockchain.NewAPI(nil, nil)
```

Comment:
Is it really needed two Nils?

Reply:
nils are for applying default params inside the blockchain API constructor. 

( this API constructor is made by @Sokel, seems it must be rewritten using `options`,  like gRPC do.)

Reply:
@noxiouz, @sshaman1101 good idea, thanks